data_science/terraform/stack/notebook/main.tf

```diff
@@ -0,0 +1,34 @@
+resource "aws_route53_zone" "wecodev" {
+  name = "wecodev.com"
+}
+
+resource "aws_route53_record" "notebook" {
+  zone_id = "${aws_route53_zone.wecodev.zone_id}"
+  name    = "notebook.wecodev.com"
+  type    = "CNAME"
+  ttl     = "300"
+  records = ["${module.dlami.public_dns}"]
+}
+
+module "dlami" {
+  source = "../../modules/dlami"
```

Comment:
It feels slightly odd that this module is actually an EC2 instance not an AMI. But, just a thought there.